<launch>

   
    <arg name="K_v" default="0.5" />
    <arg name="K_omega" default="1" />
    <arg name="K_rho" default="0.2" />
    <arg name="K_alpha" default="2" />
    <arg name="K_beta" default="-0.8" />

   
    <node name="turtle_control" 
        pkg="turtle_infnet" 
        type="turtle_control.py"
        args="$(arg K_v) $(arg K_omega) $(arg K_rho) 
        		$(arg K_alpha) $(arg K_beta)"
        output="screen">

	<remap from="goal" to="/goal"/>
	<remap from="pose" to="/turtle1/pose"/>
	<remap from="cmd_vel" to="/turtle1/cmd_vel"/>
	<remap from="control_type" to="/turtle1/control_type"/>
	</node>
	
</launch>
